__tests__/app.test.js

```diff
@@ -57,3 +57,37 @@ describe.only('GET /api/articles task 4',() => {
         })
     })
 });
+
+//5
+describe('GET /api/articles/:article_id task 5',() => {
+    test('responds with status code 200, appropriate object properties and values',() => {
+        const reqarticle_id=1
+        return request(app).get(`/api/articles/${reqarticle_id}`)
+        .expect(200)
+        .then(({body}) =>{
+            expect(...body.article).toEqual(
```

Comment:
We don't need to worry about spreading here as we aren't concerned with mutation within our tests.
Also would be better to explicitly check for values rather than any String/Number. We know what the response body should look like with that specific id so we can hard code these values.